src/CTA.WebForms/FileConverters/CodeFileConverter.cs

```diff
@@ -42,11 +42,11 @@ public class CodeFileConverter : FileConverter
             try
             {
                 _fileModel = _webFormsProjectAnaylzer.AnalyzerResult.ProjectBuildResult?.SourceFileBuildResults?
-                    .Single(r => r.SourceFileFullPath.EndsWith(RelativePath))?.SemanticModel;
+                    .Single(r => r.SourceFileFullPath == FullPath)?.SemanticModel;
```

Comment:
use this for comparing strings to make sure we don't have any additional errors 

_fileModel = _webFormsProjectAnaylzer.AnalyzerResult.ProjectBuildResult?.SourceFileBuildResults?
                    .Single(r => r.SourceFileFullPath.Equals(fullPath, StringComparison.OrdinalIgnoreCase))?.SemanticModel;

Reply:
Good catch, will make that change